kymatio/scattering1d/filter_bank.py

```diff
@@ -547,10 +549,14 @@ def calibrate_scattering_filters(J, Q, r_psi=math.sqrt(0.5), sigma0=0.1,
                                             alpha=alpha)
     xi2, sigma2, j2 = compute_params_filterbank(sigma_low, 1, r_psi=r_psi,
                                             alpha=alpha)
+    ## PAW
+    #sigma_low = sigma_low * math.pow(2, 5)
+    sigma_low = sigma0/T
```

Comment:
it's a bit confusing that sigma_low is defined twice in this function: once for calling compute_params_filterbank and a second time as a return value. Perhaps we should clarify what's going on here?

Reply:
I have clarified this in the coming revision